tap_gitlab/__init__.py

```diff
@@ -51,6 +51,7 @@
     },
 }
 
+#pylint: disable=invalid-name
```

Comment:
I would actually recommend changing these globals to all caps. That seems like a reasonable standard. @iterati and I have made that change in a couple other taps.